bears/general/LineContinuationBear.py

```diff
@@ -46,4 +49,15 @@ def run(self,
                         column=len(line) - 1,
                         end_line=line_number + 1,
                         end_column=len(line),
-                        )
+                    )
+            elif len(line) > 1 and ignore_with:
+                if line[-2] in line_continuation and line[:5] != 'with ':
```

Comment:
`line[:5]` assumes the position of `with` in the source line, which is not acceptable as with can appear at any indentation.